source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss

```diff
@@ -149,7 +150,7 @@ $font-family: 'Segoe UI', sans-serif;
         text-overflow: ellipsis;
         text-align: center;
         vertical-align: middle;
-        cursor: default;
+        cursor: pointer;
```

Comment:
can we check with designer? As I know, in xaml, we used backgroundcolor/bordercolor to signal the hover other than pointer. Also I'm wondering if it introduces issue on xbox.

Reply:
![image](https://user-images.githubusercontent.com/21356912/160946519-ec730c7b-967c-4d5e-9f4e-e69b8a3ca49a.png)

Reply:
Add @paulcam206 the question here is that should we take the pointer change?

Reply:
I think the change is fine as-is. 

I don't know think we want to apply a default hover color -- there's no generically-correct color to use. It looks like some hosts already define one in CSS (note that this particular style wouldn't work correctly if the rest of its theme were dark-mode oriented):

https://github.com/microsoft/AdaptiveCards/blob/3a35997792bd7f7b4d4deb155f21804803084e70/source/nodejs/adaptivecards-designer/src/containers/outlook/outlook-container.css#L60-L66